### ISSUE TITLE
Parsing arguments of nested fields

### ISSUE THREAD

**Issue by apoggi-carecloud** [robrichard/graphql-fields#10]
I would suggest adding an option to parse out the arguments passed on nested fields.

**Comment by nicolas-marien** [comment]
That would be an awesome addition to this utils. What would the output be like ?
Would something like the following be a acceptable output ? With value being _stringyfied_ for instance.

```json
{
  "fields": {
    "subfields (arg: value)": {}
  }
}
```

or

```json
{
  "fields": {
    "subfields": {
      "__arguments": [
        { [argName]: { kind: 'EnumValue', value: 'MY_ENUM_ENTRY' } }
      ]
    }
  }
}
```

**Comment by robrichard** [comment]
I'm open to accepting a PR that implements this. I would prefer the second alternative as parsing out the argument names would not be convenient.

The `__arguments` field would have to be added only when an additional flag is used as always returning it would break users who do `Object.keys(fields.subfields)` to test if any subfields are used..

**Comment by nicolas-marien** [comment]
I would happilly submit a PR for the second proposition :)

**Comment by robrichard** [comment]
Published 1.2.0
